api/src/main/java/org/pmiops/workbench/workspaces/WorkspacesController.java

```diff
@@ -338,6 +350,11 @@ private void setDbWorkspaceFields(
     // The version asserted on save is the same as the one we read via
     // getRequired() above, see RW-215 for details.
     dbWorkspace = workspaceService.saveWithLastModified(dbWorkspace);
+
+    final Workspace editedWorkspace = workspaceMapper.toApiWorkspace(dbWorkspace, fcWorkspace);
+
+    workspaceAuditAdapter.fireEditAction(
```

Comment:
Target Property is always a DB ID (if one exists).

The adapter functions take in a mixture of DB and API models as needed depending on the injection site. The goal there is always to be minimally intrusive.

Reply:
Hmm OK. I was worried that interpreting the audit events might get confusing if its not clear which model its referring to with `TARGET_PROPERTY` but I guess it's OK if our API/DB models stay consistent with each other so that it doesn't matter.